aiofcm/client.py

```diff
@@ -7,9 +7,12 @@
 
 
 class FCM:
-    def __init__(self, sender_id, api_key, max_connections=10, loop=None):
+    def __init__(self, sender_id, api_key, callback=None, min_connections=None, max_connections=10, loop=None):
```

Comment:
Let's rename `callback` to `upstream_callback`.

Reply:
Fixed